R/query-string.R

```diff
@@ -70,9 +70,9 @@ createPathRegex <- function(pathDef){
 }
 
 typeToRegex <- function(type){
-  re <- rep("[^\\\\./]+", length(type))
-  re[type == "int"] <- "\\\\d+"
-  re[type == "double" | type == "numeric"] <- "\\\\d*\\\\.?\\\\d*"
+  re <- rep("[^/]+", length(type))
+  re[type == "int"] <- "-?\\\\d+"
+  re[type == "double" | type == "numeric"] <- "-?\\\\d*\\\\.?\\\\d*"
```

Comment:
Unrelated to your code, but I believe this is actually a bug, as it matches `""` as currently written. Do you mind swapping one of the `\\\\d*`s with a `\\\\d+` so that we can be sure that there's at least one digit in the string (as we do with int)?

Reply:
Ok, you're right. I'll use `-?\d*\.?\d+` which would match e.g. `.23`, which can be interesting, I guess.